components/massmint/EditPanel.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <NeoSidebar
+    fullheight
+    right
+    overlay
+    class="sidebar-width"
+    position="fixed"
+    :open="open"
+    :can-cancel="true"
+    :on-cancel="closePanel">
+    <div
+      class="border-left theme-background-color navbar-margin p-5 is-flex is-flex-direction-column is-align-items-center is-justify-content-space-between h-full">
+      <div
+        class="is-flex w-full is-flex-direction-column is-justify-content-space-between is-align-items-center">
+        <div class="is-flex w-full">
+          <div class="is-flex is-justify-content-center is-flex-grow-1">
+            {{ $t('massmint.edit') }} #{{ nft?.id }}
+          </div>
+          <NeoIcon
+            icon="close"
+            size="small"
+            class="is-clickable"
+            @click.native="closePanel" />
```

Comment:
click on icon?

Reply:
can change to NeoButton i guess

Reply:
> can change to NeoButton i guess

*Should change, better accessibility and no side effect stuff

Reply:
aight

mind you, it is NeoIcon in connectWallet and notifications modals as well

Reply:
> mind you, it is NeoIcon in connectWallet and notifications modals as well

Sorry I can't be on all reviews 😭. if you can't, it's okay, we'll see later